src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -1132,7 +1132,7 @@ public WrapModeType WrapModeV
         /// If this is set to be true, then the width or height, which is not set by user explicitly, can be adjusted to preserve the aspect ratio of the image resource.
```

Comment:
do you mean " can be adjusted while maintaining the aspect ratio"?

Reply:
It means the ImageView's width or height value can be changed to preserve the aspect ratio of image resource.
I will update the documentation to make users understand better.